src/filesystem/impls/filer/lib/content.js

```diff
@@ -87,7 +87,6 @@ define(function (require, exports, module) {
         case '.mp3':
             return 'audio/mpeg';
         case '.wave':
-        // fallsthrough
```

Comment:
Everywhere that we fall through from one case to the next, we need this.  Can you add back, and also add on 87 above?